### PR TITLE
updated tree examples to use git log --graph output

### PR DESCRIPTION
**Pull request by jzarakas**
Not sure I dig the formatting, but might be less confusing.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -285,19 +278,16 @@
...
+                *   b7df983 added agenda
+                |\  
+                | * 6936cfc added some definitions
+                * | 64d856e fixed typos
+                |/  
+                * 3bc4271 merged branch
               </code>
             </pre>
-            <p>(6) contains changes between (5) and (2) merged into (3)</p>
+            <p>(3bc4271) contains changes between (6936cfc) and (6936cfc) merged together</p>
```
Comment (vmaliwal) [inline index.html]: I am not sure if this is easier to grasp while talking, esp. with the hash.

File: index.html [hunk 2]
```diff
@@ -215,11 +215,9 @@
             <p>When intializing a git repo the default branch is master</p>
             <pre>
               <code>
-                (1) -- (2) -- (3)
-                               |
-                             master
-                               |
-                              HEAD
+               HEAD -->  * fe1e7c5 Updated image 1
```
Comment (jzarakas) [inline index.html]: @vmaliwal  does this notation work for you?

File: index.html (hunk 1 above)
Comment (jzarakas) [inline index.html]: Does the notation on Line #218 work for you? (HEAD-->)

Comment (vmaliwal) [inline index.html]: Sure that works.

**Comment by vmaliwal** [comment]
Thanks

**Comment by jzarakas** [comment]
You're welcome.

**Pull request merged**